Mathlib/Data/Nat/Digits.lean

```diff
@@ -370,6 +373,28 @@ theorem getLast_digit_ne_zero (b : ℕ) {m : ℕ} (hm : m ≠ 0) :
     exact Nat.div_pos (le_of_not_lt hnb) (zero_lt_succ (succ b))
 #align nat.last_digit_ne_zero Nat.getLast_digit_ne_zero
 
+theorem mul_ofDigits (n : ℕ) {b : ℕ} {l : List ℕ} :
+    n * ofDigits b l = ofDigits b (l.map (n * ·)) := by
+  induction l with
+  | nil => rfl
+  | cons hd tl ih =>
+    rw [List.map_cons, ofDigits_cons, ofDigits_cons, ← ih]
+    ring
+
+theorem ofDigits_add_ofDigits_eq_ofDigits_zipWith_of_length_eq (b : ℕ) (l1 l2 : List ℕ)
```

Comment:
That's a very long name. Is there something else that ofDigits_add_ofDigits could be that you need to distinguish it from?